config/__init__.py

```diff
@@ -0,0 +1,14 @@
+from os import environ
+
+FLASK_ENV = environ.get("FLASK_ENV")
+
+if FLASK_ENV == "development":
```

Comment:
Could we use the pattern matching introduced in PEP-634 e.g. https://peps.python.org/pep-0636/#appendix-a ?

Reply:
A discussion for another time but I would advocate for a longer line length if it makes for more ledgible code, e.g. on Digital Marketplace we did this: https://github.com/Crown-Commercial-Service/digitalmarketplace-api/blob/ec80156ba778d64ed0dd42de0fe3a0bff98e3cf9/.flake8#L11